tests/util/misc.py

```diff
@@ -116,6 +116,20 @@ def quadratic(x: tf.Tensor) -> tf.Tensor:
     return tf.reduce_sum(x ** 2, axis=-1, keepdims=True)
 
 
+def linear(x: tf.Tensor) -> tf.Tensor:
+    r"""
+    The multi-dimensional linear function.
+
+    :param x: A tensor whose last dimension is of length greater than zero.
+    :return: The sum :math:`x` of ``x``.
+    :raise ValueError: If ``x`` is a scalar or has empty trailing dimension.
+    """
+    if x.shape == [] or x.shape[-1] == 0:
+        raise ValueError(f"x must have non-empty trailing dimension, got shape {x.shape}")
+
+    return tf.reduce_sum(x, axis=-1, keepdims=True)
```

Comment:
I'm not clear on the benefit to making this available to all the tests. `quadratic` is a simple function with a single global minimum. Is there something `linear` offers that we'll want to use frequently?

Reply:
I was not aware of the usage of `quadratic` in that way and originally thought `misc.py` might just be the place to put a new model for use. 
It is assumed to be often used for test of multi-objective related acquisition function, the reason, as mentioned above, is it might be better to have different model for different objective function. The reason to choose linear is just heuristically thought it can provide a model that is enough different than the `QuadraticMeanAndRBFKernel` test model, and it is not ristricted to 2 dimension input as the  `GaussianProcess([branin], [kernel])`. 
But this can also be moved directly into the test for multi-objective related function if this is not useful for others, would this move a better choice at the moment?